README.md

```diff
@@ -14,11 +18,16 @@ Usage:
    <remoteUsername>@<targetIdentity> [flags]
 
 Flags:
-  -i, --SshKeyPath string   Path to ssh key. default: $HOME/.ssh/id_rsa
-  -c, --ZConfig string      Path to ziti config file. default: $HOME/.ziti/zssh.json
-  -d, --debug               pass to enable additional debug information
-  -h, --help                help for this command
-  -s, --service string      service name. default: zssh (default "zssh")
+  -p, --CallbackPort string   Port for Callback. default: 63275 (default "63275")
```

Comment:
I expected to only kebab-case long options like `--word-another`. The inconsistent capitalization is friction. We should fix that, optionally adding aliases for legacy random case options during a deprecation period. I don't feel like that's necessary, however.

Reply:
We can look into fixing this sure. not in this PR but sometime in the future